.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
+
+[*.py]
```

Comment:
Per our discussion, we probably don't need this as there won't be any Python in this project.  I don't see how it would hurt though to enforce formatting rules so I'm indifferent about removing it.

Reply:
Might as well remove it, we can always add back later.

Reply:
Agreed.